### PR TITLE
Fixed #12651: providing links to the .cum* in .expanding

### LINKED ISSUE #12651: DOC: improve cross-links in docs between .expanding and .cum*
xref #12648 
- cross-link doc-strings for .cum\* to the .expanding().\* (for sum,prod,max,min, which are only supported).
- show an example (in computation.rst wher eexpanding is shown? or basics.rst (where cum\* are shown)) how these are related (w.r.t. NaN filling).

```
In [7]: s = Series([1,2,np.nan,3,np.nan,4])

In [8]: s.cumsum()
Out[8]: 
0     1.0
1     3.0
2     NaN
3     6.0
4     NaN
5    10.0
dtype: float64

In [9]: s.expanding().sum()
Out[9]: 
0     1.0
1     3.0
2     3.0
3     6.0
4     6.0
5    10.0
dtype: float64

In [10]: s.cumsum().ffill()
Out[10]: 
0     1.0
1     3.0
2     3.0
3     6.0
4     6.0
5    10.0
dtype: float64
```

### PR DESCRIPTION
**Pull request by prabhjotsumman**
- [x] closes #12651
- [ ] tests added / passed
- [ ] passes `git diff upstream/master | flake8 --diff`
- [x] whatsnew entry
  #12651
  
  commit for the .cum\* in .expanding

### CODE REVIEW

**Comment by sinhrks** [comment]
Should close #12651 (not #12561).

I think it is better to link each other via docstring as described in #12561, or use section links ( ``..sealso::`` ). It's unclear what the updated part tries to explain.

**Comment by Prabhnith** [comment]
sectional links provided.

File: doc/source/computation.rst [hunk 1]
```diff
@@ -578,6 +578,7 @@ they are implemented in pandas such that the following two calls are equivalent:
    df.expanding(min_periods=1).mean()[:5]
 
 These have a similar set of methods to ``.rolling`` methods.
+Please see the :ref:`cumsum, cummax, cummin, cumprod <basics.stats>`
```
Comment (jreback) [inline doc/source/computation.rst]: I wold expect a ``note::`` section here explaining the differnce as I have noted in the issue

**Comment by jreback** [comment]
pls reopen if you like to update according to comments

**Pull request closed without merging**